src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -14,7 +14,7 @@ public function assertResponseOk()
 	{
 		$actual = $this->response->getStatusCode();
 
-		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' .$actual);
+		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' . $actual);
```

Comment:
@GrahamCampbell , sorry mate, that very fact sometimes makes one irritated. Not your fault, I know, but it is what it is. I know it is hard to maintain this project. But let's try to be a little bit considerate towards our contributors - it's not our aim to destroy this good project after all. Anyway, apologies if I offended.